pkg/codegen/testing/test/testdata/kubernetes20/java/src/main/java/com/pulumi/kubernetes/core/v1/ConfigMap.java

```diff
@@ -1,15 +1,15 @@
 // *** WARNING: this file was generated by test. ***
 // *** Do not edit by hand unless you're certain you know what you are doing! ***
 
-package com.pulumi.kubernetes.core_v1;
+package com.pulumi.kubernetes.core.v1;
```

Comment:
Previously the tests ignored any language specific settings. They now no longer do, and this change comes from the config

```json
    "java": {
      "packages": {
        "core/v1": "core.v1",
        "helm.sh/v3": "helm.v3",
        "meta/v1": "meta.v1"
      }
    },
```